src/main/java/com/beis/subsidy/control/publicsearchservice/service/SearchService.java

```diff
@@ -86,6 +154,21 @@ public SearchResults findMatchingAwards(SearchInput searchinput) {
 		return awardRepository.findAll();
 	}
 
+	/**
+	 * 
+	 * @param direction - direction of sort
+	 * @return Sort.Direction - sort direction
+	 */
+	private Sort.Direction getSortDirection(String direction) {
+	    
+		if (direction.equals("asc")) {
+	      return Sort.Direction.ASC;
+	    } else if (direction.equals("desc")) {
```

Comment:
if condition is not required here by default returning Sort.Direction.ASC so add  if (direction.equals("desc") condition only here.